### ISSUE TITLE
Property 'send' does not exist on type 'ACMClient'

### ISSUE THREAD

**Issue by uduncanu** [aws/aws-sdk-js-v3#4927]
### Checkboxes for prior research

- [X] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest)
- [X] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js).
- [X] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js-v3/issues) and didn't find any solution.

### Describe the bug

Since v3.363.0, we've been getting errors when trying to list ACM certificates using code based on the example at https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/acm/command/ListCertificatesCommand/

Downgrading @aws-sdk/client-acm to v3.362.0 fixes the issue, and the only change in the release notes for 3.363.0 that looks relevant is #4873.

### SDK version number

@aws-sdk/client-acm@3.363.0

### Which JavaScript Runtime is this issue in?

Node.js

### Details of the browser/Node.js/ReactNative version

v18.12.0

### Reproduction Steps

Following the example from https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/acm/command/ListCertificatesCommand/ , we're running this code:

```
    var acm = new ACMClient({ region: 'us-east-1' });
    let params: ListCertificatesCommandInput = {
      CertificateStatuses: ['ISSUED'],
    };
    let certs = await acm.send(new ListCertificatesCommand(params));
```

### Observed Behavior

This error is returned:
```
error TS2339: Property 'send' does not exist on type 'ACMClient'.

301       let certs = await acm.send(new ListCertificatesCommand(params));
```

### Expected Behavior

I was expecting to not get an error, and get a list of ACM certificates.

**Comment by RanVaknin** [comment]
Hi @uduncanu ,

Thanks for letting us know.

Unfortunately using the information provided I'm not able to reproduce the described behavior.

My code:
```ts
import { ACMClient, ListCertificatesCommand, ListCertificatesCommandInput } from "@aws-sdk/client-acm";

const acm = new ACMClient({ region: 'us-east-1' });
let params: ListCertificatesCommandInput = {
  CertificateStatuses: ['ISSUED'],
};

async function listCerts() {
  try {
      const certs = await acm.send(new ListCertificatesCommand(params));
      console.log(certs)
  } catch (error) {
      console.log(error)
  }
}
listCerts()
```

```console
$ npm list
4927@1.0.0 /Users/rvaknin/test_folder/4927
├── @aws-sdk/client-acm@3.363.0
└── @types/node@20.4.0

$ tsc sample.ts
$ node sample.js
{
  '$metadata': {
    httpStatusCode: 200,
    requestId: 'REDACTED',
    extendedRequestId: undefined,
    cfId: undefined,
    attempts: 1,
    totalRetryDelay: 0
  },
  CertificateSummaryList: [
    {
      CertificateArn: 'arn:aws:acm:us-east-1:REDACTED:certificate/REDACTED',
      CreatedAt: 2023-01-18T03:01:24.864Z,
      DomainName: 'rvaknin.people.aws.dev',
      ExtendedKeyUsages: [Array],
      HasAdditionalSubjectAlternativeNames: false,
      InUse: true,
      IssuedAt: 2023-01-18T03:01:47.566Z,
      KeyAlgorithm: 'RSA-2048',
      KeyUsages: [Array],
      NotAfter: 2024-02-17T23:59:59.000Z,
      NotBefore: 2023-01-18T00:00:00.000Z,
      RenewalEligibility: 'ELIGIBLE',
      Status: 'ISSUED',
      SubjectAlternativeNameSummaries: [Array],
      Type: 'AMAZON_ISSUED'
    }
  ]
}
```

Can you please upload a minimal Github repository so that we can clone and test it ourselves? 

Thank you!
Ran~

**Comment by uduncanu** [comment]
Hi @RanVaknin!

I think this might be linked to the version of TypeScript. I'm playing around with the code you used and get the errors with TypeScript 3.9.4, but not with 4.9.3 or 5.1.6.

It's also worth noting that we're actually running this as part of an AWS CDK project, which created package.json for us via `cdk init --language typescript`.  From what I can tell, once that file has been created nothing on the CDK side ever updates it. The [upgrade from v3 to v4](https://github.com/aws/aws-cdk/commit/c9fdc8ad992669ac907e21e26bd8d197cfbc86c7#diff-ea92fa2036604eec3a3e3cafaa2b0a34e1a58a50f2d29c39d060e1650cabc701) happened in early December last year, and we created the project before then.

Using the same code you used and this package.json file, I can replicate the error by running `npm run build sample.ts`.

```
{
  "name": "bug-4927",
  "version": "0.1.0",
  "scripts": {
    "build": "tsc"
  },
  "devDependencies": {
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "@aws-sdk/client-acm": "^3.363.0"
  }
}
```

**Comment by RanVaknin** [comment]
Hi @uduncanu ,

We merged https://github.com/awslabs/smithy-typescript/pull/811. 

Can you tell us if this solves your issue?

Thanks,
Ran~

**Comment by Naddiseo** [comment]
> Hi @uduncanu ,
> 
> We merged [awslabs/smithy-typescript#811](https://github.com/awslabs/smithy-typescript/pull/811).
> 
> Can you tell us if this solves your issue?
> 
> Thanks, Ran~

I upgraded my `@aws-sdk/*` packages to 3.369 which seems to include `@smithy/types` 1.1.1 with the mentioned released, and the type problems I saw in https://github.com/aws/aws-sdk-js-v3/issues/4928 seem to be fixed. (I've also set my package.json resolutions set `@smithy/types` to 1.1.1)
